### PR TITLE
[SSAT] Remove some log files generated by SSAT tests

### PR DESCRIPTION
**Pull request by anyj0527**
- Remove some log files during SSAT test for humble yocto qemu environment

**Self evaluation:**
1. Build test: [X]Passed [ ]Failed [ ]Skipped
2. Run test: [X]Passed [ ]Failed [ ]Skipped

### CODE REVIEW

File: tests/nnstreamer_rate/runTest.sh [hunk 1]
```diff
@@ -31,28 +31,35 @@ callCompareTest original_0.log result_0.log 2-1 "Compare 2-1" 1 0
...
 gstTest "--gst-plugin-path=${PATH_TO_PLUGIN} videotestsrc num-buffers=300 ! video/x-raw,width=640,height=480,framerate=30/1 ! tensor_converter ! tensor_rate framerate=60/1 throttle=false ! multifilesink location=result_%1d.log" 5 0 0 $PERFORMANCE
 
 callCompareTest original_0.log result_0.log 5-1 "Compare 5-1" 1 0
 callCompareTest original_150.log result_300.log 5-2 "Compare 5-2" 1 0
 callCompareTest original_200.log result_400.log 5-3 "Compare 5-3" 1 0
 
-rm original_*.log
 rm result_*.log
 
+rm original_*.log
```
Comment (jaeyun-jung) [inline tests/nnstreamer_rate/runTest.sh:63]: Q) why do you move this line?

Comment (anyj0527) [reply tests/nnstreamer_rate/runTest.sh:63]: Oh I think I did it for consistency of `rm result_*.log` after `callCompareTest`

File: tests/nnstreamer_converter/runTest.sh [hunk 2]
```diff
@@ -84,4 +84,6 @@ gstTest "--gst-plugin-path=${PATH_TO_PLUGIN} audiotestsrc num-buffers=1 samplesp
 gstTest "--gst-plugin-path=${PATH_TO_PLUGIN} multifilesrc location=\"testsequence_%1d.png\" index=0 caps=\"image/png,framerate=\(fraction\)30/1\" ! pngdec ! videoconvert ! tensor_converter ! filesink location=\"testcase08.log\"" 6-1 0 0 $PERFORMANCE
 callCompareTest testcase08.golden testcase08.log 6-2 "PNG Stream Test" 0 0
 
+rm *.log
```
Comment (gichan-jang) [inline tests/nnstreamer_converter/runTest.sh:87]: Is it okay not to remove *.golden, *.png or etc.?

Comment (anyj0527) [reply tests/nnstreamer_converter/runTest.sh:87]: User may give `SKIPGEN` option for SSAT after first run. I think removing *.log is enough.

**Review by gichan-jang — APPROVED** [review]

**Review by myungjoo — APPROVED** [review]
LGTM

**Pull request merged**